src/graphics/resources.rs

```diff
@@ -21,12 +26,15 @@ pub struct Resources {
 impl Resources {
     pub fn new(image: &Image, client: &Client) -> Self {
         let target_texture = Texture::new_target(client);
+        let _source_texture = Texture::new_source(image, client);
```

Comment:
A variable starting with `_` means that it's unused (in Rust). Did you want to avoid a warning, or you forgot to remove it?

Reply:
It's a special case. The texture needs to be saved so the memory stays allocated, but we don't use this variable after its declaration.

Reply:
Ok, ok, in that case, can you please add an explanation above this member in the struct?